### PR TITLE
Updates Quick Start examples to run as expected

### PR DESCRIPTION
**Pull request by kennethormandy**
Hey! Thanks so much for the hard work on this project. I’m just making my way through the documentation now, and I couldn’t get the Quick Start examples to run until I made the following changes.

I have very little React experience, and even less with ES2015 stuff—it’s entirely possible I just misunderstood something in all these cases. Regardless, the examples didn’t work as-is for me, and to the best of my understand, these changes fix that.

The one change I made that was maybe more of an opinion was turning:

```js
const {editorState} = this.state;
return <Editor editorState={editorState} onChange={this.onChange} />
```

…into:

```js
return <Editor editorState={this.state.editorState} onChange={this.onChange} />
```

…which just reduced the complexity for me at the beginning, and had more in common with the initial Input example.

### CODE REVIEW

File: docs/QuickStart-API-Basics.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ to provide information about the text that the user has written.
 class MyInput extends React.Component {
   constructor(props) {
     super(props);
+    this.state = { value: '' }
```
Comment (hellendag) [inline docs/QuickStart-API-Basics.md]: this.state = {value: ''};

**Comment by hellendag** [comment]
You're right about the missing state initialization, that was an oversight on my part. Nice catch.

I don't think it's so bad to have the destructuring assignment, especially since we're using ES6 classes and even `import` syntax here. But since the goal of these docs is to get people up to speed quickly, I don't want anyone stressing about syntax too much. So I'm cool with the change.

File: docs/QuickStart-Rich-Styling.md [hunk 2]
```diff
@@ -44,15 +44,16 @@ We can observe and handle key commands via the `handleKeyCommand` prop, and
...
 class MyEditor extends React.Component {
   constructor(props) {
     super(props);
+    this.state = { editorState: EditorState.createEmpty() }
     this.onChange = (editorState) => this.setState({editorState});
   }
   handleKeyCommand(command) {
-    const {editorState} = this.state;
-    const newState = RichUtils.handleKeyCommand(editorState, command);
+    const newState = RichUtils.handleKeyCommand(this.editorState, command);
```
Comment (tasti) [inline docs/QuickStart-Rich-Styling.md]: `this.editorState` should be `this.state.editorState`

**Comment by kennethormandy** [comment]
Thanks for the review. I made the changes you suggested, and change one more instance of:

```js
const {editorState} = this.state;
this.onChange(RichUtils.toggleInlineStyle(editorState, 'BOLD'));
```

…to:

```js
this.onChange(RichUtils.toggleInlineStyle(this.state.editorState, 'BOLD'));
```

…for consistency with the other places I changed that in the Quick Start guide.

**Comment by hellendag** [comment]
Looks good, thanks.

**Comment by hellendag** [comment]
@facebook-github-bot import

**Pull request closed without merging**